Controllers/conversationController.js

```diff
@@ -108,3 +110,36 @@ exports.show = CatchAsyncError(async (req, res, next) => {
     })
   }
 })
+
+exports.createConversation = CatchAsyncError(async (req, res, next) => {
+  // Find searched user
+  const searchedUser = await User.findOne({
+    raw: true,
+    attributes: ['id'],
+    where: {
+      username: req.body.searchedUserUsername
+    }
+  })
+  // Check if conversation already Exists
+  if (req.body.usersId.includes(searchedUser.id)) {
```

Comment:
This only checks if the user you want to create a conversations with exists, not that you have an existing conversation with that user.